backend/internal/db/dbgorm/deployments_test.go

```diff
@@ -0,0 +1,122 @@
+package dbgorm
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"go.autokitteh.dev/autokitteh/backend/internal/db/dbgorm/scheme"
+	"go.autokitteh.dev/autokitteh/sdk/sdkservices"
+	"go.autokitteh.dev/autokitteh/sdk/sdktypes"
+	"gorm.io/gorm"
```

Comment:
Our convention is to have (up to) 3 separate blocks inside `import()`: standard Go packages, non-AK, and AK. i.e. move gorm to the 2nd block